src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1311,7 +1323,7 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
 
       endif
 
-      if ( if_model_dbz )then
+      if ( if_model_dbz .or. if_model_fed )then
```

Comment:
Simlarily, those condition could be represented by  "if nphyvario3d>0) "

Reply:
@TingLei-NOAA 
Can this line leave as it is? I think it makes the codes more readable as it shows it is related to dbz and fed DA even a user reads code first time, and the style be similar to the coded in the sections around.  For examples: 
  if (laeroana_fv3cmaq) then,     
 if (laeroana_fv3smoke) then).

Reply:
In fact, I am thinking if those la*cmaq/smoke could be simplified.  but, it is ok for the current PR.